__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const request = require('supertest');
+const app = require('../app')
+const seed = require('../db/seeds/seed')
+const data = require('../db/data/test-data')
+const db = require('../db/connection')
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+
+describe('/api/topics', () => {
+    test('GET: 200 sends an array of topics', () => {
+        return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((result) => {
+                expect(result.body.topics).toBeInstanceOf(Array);
+            })
+    })
+    test('GET: 200 each topic should have "slug" and "description" properties', () => {
+        return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((result) => {
+                const topics = result.body.topics;
+                expect(topics.every(topic => 'slug' in topic && 'description' in topic)).toBe(true)
```

Comment:
Nice work, I recommend adding a test for the length of the array as we know what the data that is being used to seed looks like.